### PR TITLE
Jenkinstest

### PR DESCRIPTION
**Pull request by kavinbala96**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,27 +1,78 @@
+@Library('pipeline-library') _
 pipeline {
+    agent {
+        node{
+            label "dev_mesos_large_v2"
+        }
+    }
+
+    environment {
+        CREDENTIALS_ID  = "GENESYSCLOUD_OAUTHCLIENT_ID_AND_SECERET"
```
Comment (carnellj-genesys) [inline Jenkinsfile:10]: SECERT=>SECRET

Comment (kavinbala96) [reply Jenkinsfile:10]: Will check with DK.

Comment (kavinbala96) [reply Jenkinsfile]: fixed

File: Jenkinsfile [hunk 2]
```diff
@@ -1,27 +1,78 @@
...
+        GOPATH = "$HOME/go"
+		TF_ACC = "1"
+        TF_LOG = "DEBUG"
+        TF_LOG_PATH = "../test.log"
+		GENESYSCLOUD_REGION = "us-east-1"
+        GENESYSCLOUD_SDK_DEBUG =  "true"
+        GENESYSCLOUD_TOKEN_POOL_SIZE =  20
+    }
+    tools {
+        go 'Go 1.20'
```
Comment (dginty4) [inline Jenkinsfile:20]: We are using go 1.21 in the provider, should we use the same here?

File: Jenkinsfile [hunk 3]
```diff
@@ -1,27 +1,78 @@
...
-    }
-  }
+
+        stage('Upload artifacts') {
+            steps {
+                archiveArtifacts artifacts: 'coverage.html', allowEmptyArchive: true
+
+            }
+        }
+
```
Comment (charliecon) [inline Jenkinsfile:75]: Is the release stage supposed to be gone? I think it is missing from the file now

File: Jenkinsfile (hunk 2 above)
Comment (kavinbala96) [reply Jenkinsfile:20]: 1.21 isn't available as of yet in Jenkins.

**Review by HemanthDogiparthi12 — APPROVED** [review]

**Pull request merged**